.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,46 @@
-## Purpose
-<!--- Describe the problems, issues, or needs driving this feature/fix and include links to related issues -->
-The purpose of this PR is to fix #<issue-number>
+---
+labels: pending-review
```

Comment:
can't we use a git label for this? But I think this is also good.

Reply:
This section act as an instruction for Github and it won't be visible to the user. It should automatically add the given label and assign provided users to the PR. Haven't tested it for the PRs yet, but I have tested it on issue templates. 

Check this: https://github.com/isaacs/github/issues/1252